app/scripts/modules/core/src/pipeline/service/execution.service.ts

```diff
@@ -343,7 +374,7 @@ export class ExecutionService {
           application.executions.data.push(re);
         }
       });
-      if (updated) {
+      if (updated && !application.executions.reloadingForFilters) {
```

Comment:
(this is the `mergeRunningExecutionsIntoExecutions` method)

If we are in the middle of reloading the filters, the `dataUpdated` call will clear the `reloadingForFilters` flag, which we don't want, as it appears the data is fully refreshed and, if the filters were just cleared, it'll show "no executions found for this pipeline" messages for any newly added pipelines.